### ISSUE TITLE
No grab available

### ISSUE THREAD

**Issue by Ruangq** [NVlabs/contact_graspnet#22]
I have a problem. When I use the model I trained to predict, the success rate of grabbing is very low, about 0.003.
I hope you can give me some help to solve this problem.

**Comment by Ruangq** [comment]
My environment is tensorflow2.4，cuda11.1, python3.7

**Comment by MartinSmeyer** [comment]
Do you run your training on a Nvidia 30XX GPU? In that case, you might try the environment posted in #19

**Comment by xlim1996** [comment]
@Ruangq Hi, you can try my environment file in #19. In my case is that TensorFlow 2.4 and 2.6 didn't work on my laptop. only TensorFlow 2.5 works well with cuda11.2 /11.3 on my laptop. Btw, don't use conda to install TensorFlow.

**Comment by Ruangq** [comment]
@xlim1996 
Hello, I showed up during the training.
![image](https://user-images.githubusercontent.com/105182704/178918891-eb448a6a-f0b6-4159-801a-1f84547b3fb6.png)
But the training can be carried out normally and the loss is reduced normally.
my GPU is A40

**Comment by Ruangq** [comment]
Will this affect the training results and how to solve this problem? @xlim1996

**Comment by xlim1996** [comment]
@Ruangq 
hi, 
I didn't retrain the model. So I'm unable to answer your question. 
But the environment file works well for using contact graspnet.  You may need to ask others who have experience of the retraining model.
